scheduler/src/cook/mesos/schema.clj

```diff
@@ -945,7 +945,17 @@ for a job. E.g. {:resources {:cpus 4 :mem 3} :constraints {\"unique_host_constra
                          prior-state (:job/state job)]
                      (if (= prior-state :job.state/completed)
                        [[:db/add j :job/state :job.state/waiting]]
-                       []))}}])
+                       []))}}
+   {:db/id (d/tempid :db.part/user)
+    :db/ident :ensure-not-exists
```

Comment:
It is datomic best practices to namespace idents. Perhaps `:entity/ensure-not-exists`